### PR TITLE
Add notable changes section

### PR DESCRIPTION
**Pull request by marcosbc**
*Description of changes:* Added a "Notable changes" section, which will allow us to communicate important changes affecting the current solution to users.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by marcosbc** [comment]
cc @andresbono @beltran-rubo

File: README.md [hunk 1]
```diff
@@ -20,3 +20,24 @@ For architectural details, best practices, step-by-step instructions, and custom
...
+  The */etc/logrotate.d/com.bitnami.php* configuration file should look like this:
+
+      /opt/bitnami/php/logs/*.log {
+        weekly
+        rotate 150
+        dateext
+        compress
+        copytruncate
+        missingok
+
```
Comment (andresbono) [inline README.md]: Should this empty line be here?

Comment (marcosbc) [reply README.md]: Currently the file includes the line. It is not ideal, but we kept it for now.

File: README.md [hunk 2]
```diff
@@ -20,3 +20,24 @@ For architectural details, best practices, step-by-step instructions, and custom
 
 To post feedback, submit feature ideas, or report bugs, use the **Issues** section of this GitHub repo.
 If you'd like to submit code for this Quick Start, please review the [AWS Quick Start Contributor's Kit](https://aws-quickstart.github.io/). 
+
+## Notable changes
+
+### 5.2.3-0 (2019/09/08)
+
```
Comment (andresbono) [inline README.md]: What if we add a reference to the commit id where the change was introduced?
```suggestion

https://github.com/aws-quickstart/quickstart-bitnami-wordpress/commit/511126369dc4e543c3cc12df17e8ebb3836a628e
```

**Comment by andrew-glenn** [comment]
Nice!

@andresbono @marcosbc - This potentially opens the door to release tags (with a changelog?) on the master branch. Is that something y'all would be open to?

**Comment by andrew-glenn** [comment]
Assuming yes - let's get an email thread going so we can work on the details offline.

**Pull request merged**